src/reactor.js

```diff
@@ -285,6 +290,21 @@ class Reactor {
       this.__isDispatching = false
     }
   }
+
+  /**
+   * Retrieve cache values
+   * @returns {Immutable.Map}
+   */
+  getCacheValues() {
+    return this.reactorState.get('cache')
+  }
+
+  /**
+   * Clear all cached values
+   */
+  clearCacheValues() {
+    this.reactorState = this.reactorState.set('cache', Immutable.Map())
```

Comment:
added as an afterthought, thought it might be useful in some cases of switching views in a spa. will remove